_posts/2023-08-29-data-donuts-37.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+
+#event information
+title:  "Customer Experience with LA Metro"
+date:   2023-08-29 08:00+0800
+start_time: "08:00am"
+end_time: "10:00am"
+
+location: LACI
+cover: ../images/la-kretz-campus.jpeg
+
+#event speaker details
+speakers: 
+ - "Lauren Deaderick"
+eventbrite: "https://august-2023-data-donuts.eventbrite.com"
```

Comment:
This is used for the embedd code, I don't believe this will work. You need the unique event id, which is part of the url when you are editing the event

Reply:
Will this link https://www.eventbrite.com/e/august-2023-data-donuts-registration-691215222337 work? 

I can't seem to find the link you had in the previous event.

Reply:
You'll want to use this number, see screenshot "691215222337"

![Screenshot 2023-08-03 at 4 53 37 PM](https://github.com/compilerla/data-donuts/assets/4284662/5ef0857d-4b53-4e69-9caa-0739ea3c0390)

Reply:
ahh sorry it's actually really hack-y. I use the old link and then change the id string in the middle of it. I should fix that.